### ISSUE TITLE
Sources JARs should not be downloaded

### ISSUE THREAD

**Issue by dmoklyakov** [theRookieCoder/ferium#27]
Hi Ilesh!

I found your program very useful and really enjoy using it!
But today I encountered a problem that I want to report.

## Problem
I have [this](https://github.com/PTOM76/Plant-In-A-Jar/releases/tag/v2.2.3) mod added in config:

```
        {
          "name": "Plant-In-A-Jar",
          "identifier": {
            "GitHubRepository": [
              "PTOM76",
              "Plant-In-A-Jar"
            ]
          },
          "check_game_version": false,
          "check_mod_loader": false
        },
```

And after executing `./ferium upgrade` command my "mods" folder contains "plantinajar-2.2.3-sources.jar" which is a `sources` file and they will not work.

## Possible solution
I propose to add the ability to specify a file name or regex pattern in "GitHubRepository" section of config.

**Comment by theRookieCoder** [comment]
I guess the actual problem here is that checks should exclude jars with `source` in them. Is there any other reason for regex patterns, I feel that may be pretty overkill? Also I would recommand adding the command to path if you're planning to use it!

**Comment by theRookieCoder** [comment]
> Maybe, but theoretically author can publish anything (several mod versions at one time, for example).

Right, then could you make a seperate issue for adding 'release filtering' for GitHub mods? IMO this issue more a of bug and the feature you proposed is quite different

**Comment by theRookieCoder** [comment]
Fixed in `v3.23.0` b7c74b72467f9a2981c57a9513bcf3b912e88f42
